pkg/cluster/internal/providers/podman/provider.go

```diff
@@ -396,6 +399,12 @@ func info(logger log.Logger) (*providers.ProviderInfo, error) {
 	if err := json.Unmarshal(out, &pInfo); err != nil {
 		return nil, err
 	}
+
+	// Check if there is enough disk space available
+	if err := common.CheckFreeDiskSpace(pInfo.Store.GraphRoot); err != nil {
+		return nil, err
```

Comment:
I think it would be better if kind does not takes on the system management task like checking free disk space but if it still happens I think instead of `err` this should just print a `warn` here and actual error message from provider can be relayed back to user that would be really helpful.